pkg/arvo/app/chat-cli.hoon

```diff
@@ -445,17 +445,17 @@
   ++  tab-list
     ^-  (list (option:auto tank))
     :~
-      [%join leaf+";join ~ship/chat-name (glyph)"]
-      [%leave leaf+";leave ~ship/chat-name"]
+      [%join leaf+";join (~/)~ship/chat-name (glyph)"]
+      [%leave leaf+";leave (~/)~ship/chat-name"]
       ::
       [%create leaf+";create [type] /chat-name (glyph)"]
       [%delete leaf+";delete /chat-name"]
       [%invite leaf+";invite /chat-name ~ships"]
       [%banish leaf+";banish /chat-name ~ships"]
```

Comment:
`;invite` and `;banish` actually also support both kinds of chats currently.

```suggestion
      [%invite leaf+";invite (~/)/chat-name ~ships"]
      [%banish leaf+";banish (~/)/chat-name ~ships"]
```

Reply:
Fixed.